src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -206,9 +206,12 @@ def load_arguments(self, _):
         c.argument('host_group_id',
                    validator=validate_host_group_id, is_preview=True)
         c.argument('crg_id', validator=validate_crg_id, is_preview=True)
-        c.argument('message_of_the_day', type=str)  # no validation for aks create because it already only supports Linux.
```

Comment:
why so many irrelevant changes in many files? without these changes, will Style or linter checks fail?

Reply:
linter didn't fail. And the style change auto-saved.